zip-0226.rst

```diff
@@ -298,6 +312,16 @@ Burn Mechanism Consensus Rules
 
 **Note:** Even if this mechanism allows having transparent ↔ shielded Asset transfers in theory, the transparent protocol will not be changed with this ZIP to adapt to a multiple Asset structure. This means that unless future consensus rules changes do allow it, unshielding will not be possible for Custom Assets.
 
+Rationale for Burn Mechanism
+````````````````````````````
+
+As for the value balance verification, we assume :math:`n` Actions, :math:`m` of which are related to ZEC and the remaining :math:`n-m` Actions related to Custom Assets.
+
+The burn mechanism verification is equivalent to:
+
+.. math:: \mathsf{bvk} = (\mathsf{(\sum_{i=1}^{m} cv_i^{net})} + \mathsf{(\sum_{j=m+1}^{n} cv_j^{net})}) - ([\mathsf{v^{balanceOrchard}}]\mathcal{V}^{\mathsf{Orchard}} + [0]\mathcal{R}^{\mathsf{Orchard}}) - (\mathsf{\sum_{k=1}^{L}} [\mathsf{v_{k}}]\mathsf{AssetBase_{k}} + [0]\mathcal{R}^{\mathsf{Orchard}})
```

Comment:
```suggestion
.. math:: \mathsf{bvk} = ((\sum_{i=1}^{m} \mathsf{cv^{net}}_{i}) + (\sum_{j=m+1}^{n} \mathsf{cv^{net}}_{j})) - ([\mathsf{v^{balanceOrchard}}]\mathcal{V}^{\mathsf{Orchard}} + [0]\mathcal{R}^{\mathsf{Orchard}}) - (\sum_{k=1}^{L} [\mathsf{v}_{k}]\mathsf{AssetBase}_{k} + [0]\mathcal{R}^{\mathsf{Orchard}})
```
Using normal math for set indices. Also changed the size of $\mathsf{AssetBurn}$ to normal math ($L$ instead of $\mathsf{L}$) as mentioned earlier.